src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -31,6 +31,8 @@
             <argument /> <!-- Bus handler resolver -->
         </service>
 
+        <service id="symfony.component.messenger.middleware.dispatch_after_current_bus" class="Symfony\Component\Messenger\Middleware\DispatchAfterCurrentBusMiddleware" />
```

Comment:
```suggestion
        <service id="messenger.middleware.dispatch_after_current_bus" class="Symfony\Component\Messenger\Middleware\DispatchAfterCurrentBusMiddleware" />
```

Otherwise the middleware can't be found by the short name.

Reply:
Thank you